semgrep/semgrep/util.py

```diff
@@ -5,6 +5,8 @@
 import subprocess
 import sys
 from pathlib import Path
+from itertools import chain
```

Comment:
We already have the `itertools` module imported, should just use `itertools.chain`, etc? :man_shrugging: